2016-06-03.md

```diff
@@ -0,0 +1,34 @@
+# SUBJECT
+
+## Multiple language support
+
+### Ramping up Rust
+We've long been intrigued by the Rust Language, 
```

Comment:
@tcr @johnnyman727 I'm not really sure what to say here. Tim, we had planned to write some kind of call to action for this, correct? Do you remember what?